examples/fis/main.tf

```diff
@@ -39,9 +39,9 @@ module "eks" {
   managed_node_groups = [
     {
       name          = "sockshop"
-      min_size      = 1
-      max_size      = 5
-      desired_size  = 3
+      min_size      = 2
```

Comment:
Changed the initial size of node instances to prevent scaling by cluster autoscaler. This is because k8s cluster-autoscaler confuses AWS FIS when it detects an experiment target on the currently running instance.